scripts/module_list/tests/data/lmod_mock.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Emulated avail command.
+if [ "$3" = "avail" ]; then
+  if [ "$4" = "cluster/" ]; then
+    cat "${MOCK_FILE_AVAIL_CLUSTER}" >&2
+  else
+    cat "${MOCK_FILE_AVAIL}" >&2
+  fi
+
+
+# Emulated swap command.
+elif [ "$3" = "swap" ]; then
+  #extract te cluster name from the 4th argument
+  cluster=$(echo "$4" | cut -d "/" -f 1)
+  cluster_name=$(echo "$4" | cut -d "/" -f 2)
+
+  if [ "$cluster" = "cluster" ]; then
+    # Substitute CLUSTER by the cluster_name
+    echo "${MOCK_FILE_SWAP}" | sed "s/CLUSTER/${cluster_name}/" | xargs cat >&1
+  else
+    echo "Something is wrong"
```

Comment:
explain *what* is wrong ("`Something is wrong: expected 'cluster', got '${cluster}'`")

and use `>&2` to redirect the message to stderr